server/conf/evolutions/default/81.sql

```diff
@@ -0,0 +1,9 @@
+# --- !Ups
+
+ALTER TABLE programs ADD COLUMN IF NOT EXISTS localized_short_description jsonb DEFAULT ('{"isRequired": true, "translations": {"en_US": ""}}')::jsonb;
+ALTER TABLE programs ADD COLUMN IF NOT EXISTS application_steps jsonb DEFAULT ('[]')::jsonb;
```

Comment:
**question:** (maybe blocking) Do I remember correctly from the TDD that these new columns are non-nullable?  Should we add `not null`?